io/pci/distro_tools.py

```diff
@@ -37,6 +37,8 @@ def setUp(self):
         '''
         self.option = self.params.get("test_opt", default='').split(",")
         self.tool = self.params.get("tool", default='')
+        self.interface = self.params.get("interface", default='')
+        self.location_code = self.params.get("location_code", default='')
```

Comment:
@vaishnavibhat This need not be an input. This can be got from avocado util pci.py

Reply:
@narasimhan-v Can I pass pci address as input and derive the location and interface name from the pci utility ?

Reply:
exactly, yes